lua/caramba/commands.lua

```
@@ -512,6 +529,79 @@ function M.setup()
     desc = "AI: Show index statistics",
   })
   
+  vim.api.nvim_create_user_command("AIListIndexes", function()
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)